src/main/webapp/app/shared/login/login.component.html

```diff
@@ -1,51 +1,54 @@
-    </button>
-</div>
-<div class="modal-body">
-    <div class="row justify-content-center">
-        <div class="col-md-12">
-            <div class="alert alert-danger" *ngIf="authenticationError && !captchaRequired">
-                <span class="bold">Failed to sign in!</span> Please check your username and password and try again.
-            </div>
-            <div class="alert alert-info" *ngIf="authenticationAttempts >= 3 && !captchaRequired">
-                Seems like you are having issues signing in :-( Please go to <a href="https://jirabruegge.in.tum.de">JIRA</a> and try to sign in there.
-                After that, try again here.
-            </div>
-            <div class="alert alert-danger" *ngIf="captchaRequired">
-                <span class="bold">You have entered your password too often incorrectly :-(</span><br>
-                Please go to <a href="https://jirabruegge.in.tum.de/login.jsp" target="_blank">JIRA</a>: <a href="https://jirabruegge.in.tum.de/login.jsp" target="_blank">https://jirabruegge.in.tum.de/login.jsp</a>.
-                Sign in with your account there and solve the <a href="https://jirabruegge.in.tum.de/login.jsp" target="_blank">CAPTCHA</a>.<br>
-                After you have solved the <a href="https://jirabruegge.in.tum.de/login.jsp" target="_blank">CAPTCHA</a>, try again here.
-            </div>
+<div class="row">
+    <div class="col-md-6 offset-md-3">
+        <div class="modal-header">
+            <h4 class="modal-title" jhiTranslate="login.title">Sign in</h4>
         </div>
-        <div class="col-md-12">
-            <form name="loginForm" class="form" role="form" (ngSubmit)="login()">
-                <div class="form-group">
-                    <label for="username" jhiTranslate="global.form.username">Login</label>
-                    <input type="text" class="form-control" autocomplete="username" name="username" id="username" placeholder="{{'global.form.username.placeholder' | translate}}"
-                           pattern="^([a-z]{2}\d{2}[a-z]{3})$" [(ngModel)]="username" #usernameForm="ngModel" [ngModelOptions]="{updateOn: 'blur'}">
-                    <div class="help-block" *ngIf="usernameForm.errors && (usernameForm.dirty || usernameForm.touched)">
-                        <p class="text-primary small">If you are a TUM student, your username should have the format <b>ab12xyz</b>.<br>
-                            Do <b>not</b> include '<b>@mytum.de</b>' or '<b>@tum.de</b>'.</p>
+        <div class="modal-body">
+            <div class="row justify-content-center">
+                <div class="col-md-12">
+                    <div class="alert alert-danger" *ngIf="authenticationError && !captchaRequired">
+                        <span class="bold">Failed to sign in!</span> Please check your username and password and try again.
```

Comment:
Should we use this chance to also add translations?

Reply:
done